packages/network-controller/src/NetworkController.ts

```diff
@@ -31,6 +26,11 @@ import {
 } from '@metamask/utils';
 import { INFURA_BLOCKED_KEY, NetworkStatus } from './constants';
 import { projectLogger, createModuleLogger } from './logger';
+import {
```

Comment:
Rearranged these imports slightly to better match the extension, but the new lines are really here.